cdap-security/src/main/java/co/cask/cdap/security/server/ExternalAuthenticationServer.java

```diff
@@ -98,7 +96,6 @@ public ExternalAuthenticationServer(CConfiguration configuration, SConfiguration
                                       DiscoveryService discoveryService,
                                       @Named("security.handlers") Map<String, Object> handlers,
                                       @Named(NAMED_EXTERNAL_AUTH) AuditLogHandler auditLogHandler) {
-    this.announceAddress = configuration.get(Constants.Security.AUTH_SERVER_ANNOUNCE_ADDRESS);
```

Comment:
I think it's better to have a bind.address config as well so that user can optionally control which interface to bind to.